apps/ui/src/components/MessageErrorFetchPower.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+defineProps<{
+  type: 'proposition' | 'voting';
+}>();
+
+defineEmits<{
+  (e: 'fetch');
+}>();
+</script>
+
+<template>
+  <div class="flex flex-col gap-3 items-start" v-bind="$attrs">
+    <UiAlert type="error">
+      There was an error fetching your {{ type }} power.
```

Comment:
Not too important but admins may not know what is "proposition power" (not in settings/docs) maybe we can write "There was an error fetching your validation"?

Reply:
Seems a little short, maybe "There was an error fetching your proposal validation" ?